### ISSUE TITLE
Output is too verbose when using strict_mode=false

### ISSUE THREAD

**Issue by yurirocha15** [uos/rospy_message_converter#57]
I am currently using the rospy_message_converter to convert data points from a database into ROS. However, the database values have an _id field, which does not exist in the ROS message. After setting `strict_mode=false` in the `convert_json_to_ros_message` function, I get the following output:

```shell
...
ERROR - 2021-11-23 13:26:06,846 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,846 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,846 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,847 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,847 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,847 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,847 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,847 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,848 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,848 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,848 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,848 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,848 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,849 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,849 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,849 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,849 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,850 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,850 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,850 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,850 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,850 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,851 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,851 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,851 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
ERROR - 2021-11-23 13:26:06,851 - message_converter - ROS message type "MyMsg" has no field named "_id"! It will be ignored.
...
```
As I am importing thousands of data points, this becomes too verbose.

I am aware that I could filter out the "_id" field before calling the `convert_json_to_ros_message`, but this would kill the purpose of the `strict_mode=false` option.

I believe it would be good to either make this output as `debug`, or give the user the ability to change the log level.

**Comment by mintar** [comment]
Closed via #58 .

**Comment by mintar** [comment]
Ok, I've done the release: https://github.com/ros/rosdistro/pull/32379
